routes/api/comments/index.js

```diff
@@ -121,6 +121,18 @@ router.post('/', wordlist.filter('body'), (req, res, next) => {
   });
 });
 
+router.get('/user/:user_id', (req, res, next) => {
```

Comment:
This route conflicts with existing routes:

```
/api/v1/comments/:comment_id
/api/v1/comments/user/:user_id
```

It should instead be:

```
/api/v1/comments?user_id=...
```

As you are just listing comments.